descriptors/ModuleDescriptor-template.json

```diff
@@ -143,15 +144,17 @@
           ],
           "pathPattern": "/data-import/uploadDefinitions/{uploadDefinitionId}/processFiles",
           "permissionsRequired": [
-            "data-import.uploaddefinitions.files.post"
+            "data-import.uploadDefinitions.processFiles.item.post"
           ],
           "modulePermissions": [
             "change-manager.jobexecutions.get",
             "change-manager.jobexecutions.post",
             "change-manager.jobexecutions.put",
-            "mapping-metadata.get",
-            "source-storage.records.get",
```

Comment:
please, add also "source-storage.records.matching.collection.post", "source-storage.records.formatted.item.get" permissions. Their endpoints are also used upon this endpoint call and were previously protected by "source-storage.records.get"